tests/lib/rules/import-scope.js

```diff
@@ -29,67 +29,85 @@ const testCases = {
         method: {
             require: [
                 "const map = require('lodash/map')",
-                "const someOtherModule = require('some-other-module')"
+                "const someOtherModule = require('some-other-module')",
+                "const fpMap = require('lodash/fp/map')"
             ],
             import: [
                 "import map from 'lodash/map'",
-                "import someOtherModule from 'some-other-module'"
+                "import someOtherModule from 'some-other-module'",
+                "import fpMap from 'lodash/fp/map'"
             ].map(fromModule)
         },
         member: {
             require: [
                 "const {map} = require('lodash')",
                 "const {map: m} = require('lodash')",
-                "const someOtherModule = require('some-other-module')"
+                "const someOtherModule = require('some-other-module')",
+                "const {map} = require('lodash/fp')"
             ],
             import: [
                 "import {map} from 'lodash'",
                 "import {map as m} from 'lodash'",
-                "import someOtherModule from 'some-other-module'"
+                "import someOtherModule from 'some-other-module'",
+                "import {map} from 'lodash/fp'",
+                "import {map as fpMap} from 'lodash/fp'"
             ].map(fromModule)
         },
         full: {
             require: [
                 "const _ = require('lodash')",
-                "const {someMember} = require('some-other-module')"
+                "const {someMember} = require('some-other-module')",
+                "const fp = require('lodash/fp')"
             ],
             import: [
                 "import _ from 'lodash'",
                 "import * as _ from 'lodash'",
-                "import {someMember} from 'some-other-module'"
+                "import {someMember} from 'some-other-module'",
+                "import fp from 'lodash/fp'"
             ].map(fromModule)
         }
     },
     invalid: {
```

Comment:
Arguably this shouldn't be testing the invalid/true positive cases since true positives for `lodash/fp` may be more in the wheelhouse of https://github.com/jfmengels/eslint-plugin-lodash-fp

LMK and I can nix these and just test that the false positives are fixed.